knowledge/responses/human_rights_issues_are_more_important_than_veganism.md

```diff
@@ -1,9 +1,9 @@
+One of the main issues with this excuse is that it really highlights the underlying belief that humans are not animals and we exist as separate or superior to the animal kingdom. It is a very clear indictment of the speciesist mentality our society has. And beyond all of this, is that if we all turned vegan we wouldn’t just end animal exploitation, we would end some of the most important human rights issues facing our species today. Around 800 million people in thr world are living in a state of starvation due to lack of food - food that we have but that we feed to non-human animals instead so that we can eat their flesh. In fact the United States alone could feed every single one of those 800 million people with the grain that they feed to livestock animals. Furthermore, 82% of starving children live in countries where food is grown to feed livestock animals. Going vegan could end this unfair distribution of food. 
 
-If we all went vegan we would end the exploitation of migrant workers and people in poverty, who work in slaughterhouses out of necessity and have some of the highest rates of suicide, drug and alcohol abuse, PTSD, depression and anxiety in any profession. If we all went vegan we would end the exploitation of tannery workers in India, Bangladesh and other third world countries where 90% of workers die before the age of 50 and have children born heavily disabled due to all the poisonous chemicals required in the production of leather. If we all went vegan we would end the exploitation of tribal groups in the Amazon, whose communities are being uprooted and destroyed so that the animal agriculture industries can destroy more of the rainforest. 
+Furthermore, people who work in slaughterhouses out of necessity have some of the <a href="https://journals.sagepub.com/doi/full/10.1177/15248380211030243">highest rates of suicide, drug and alcohol abuse, PTSD, depression and anxiety in any profession.</a> <a href="https://www.theguardian.com/environment/2018/jul/05/amputations-serious-injuries-us-meat-industry-plant">The industry with the second most work accidents is meat processing</a>, losing only to <a href="https://www.thesafetymag.com/ca/topics/safety-and-ppe/industys-high-death-rate-unacceptable-and-preventable/315770#:~:text=The%20data%20revealed%20fishing%20to,the%20most%20dangerous%20in%20Canada.">fisheries</a>. If we all went vegan we would end the exploitation of tannery workers in India, Bangladesh and other third world countries where 90% of workers die before the age of 50 and have children born heavily disabled due to all the poisonous chemicals required in the production of leather. If we all went vegan we would end the exploitation of tribal groups in the Amazon, whose communities are being uprooted and destroyed so that the animal agriculture industries can destroy more of the rainforest. And we would root out one of the last remaining sources of worker abuse in developed countries.
```

Comment:
Let's pull out these inline links to a "References" section below. Similar to [this response](https://github.com/veganhacktivists/animalsupportbot/blob/master/knowledge/responses/animals_eat_other_animals.md)